packages/concerto-core/lib/modelutil.js

```diff
@@ -114,29 +114,34 @@ class ModelUtil {
      * its name and version parts. The version of the namespace
      * (if present) is parsed using semver.parse.
      * @param {string} ns the namespace to parse
+     * @param {object} [options] optional parsing options
+     * @param {boolean} [options.disableVersionValidation] if false, the version will be validated
```

Comment:
How about disableVersionParsing?

Reply:
`SemVer.parse` and `SemVer.validate` are two different methods and playing around I think they are independent and validate method being the most expensive one, that's why I chose valiation.

Reply:
You're correct, however, the reason that we do validation is to handle malformed versions gracefully before parsing them. The returned object contains the parsed version by default, and that's the main client perceivable change, IMO.

Reply:
Understood, I've made the change.
If `disableVersionValidation` is set to true, it just returns the name of the model.
```
{
 name: org.accordproject.test
}
```